samples/pom.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0"?>
```

Comment:
I was trying to avoid the renaming step for the users. It would be convenient for the user to grab and use this file without any modification.